scripts/ci/jobs/check-generated.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/env bash
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")"/../../.. && pwd)"
+source "$ROOT/scripts/ci/lib.sh"
+
+set -euo pipefail
+
+go mod tidy
+
+# shellcheck disable=SC2016
+echo 'Ensure that generated files are up to date. (If this fails, run `make proto-generated-srcs && make go-generated-srcs` and commit the result.)'
+function generated_files-are-up-to-date() {
+    ln -s /go/src/github.com/stackrox /go/src/github.com/rox
+
+    git ls-files --others --exclude-standard >/tmp/untracked
+    make proto-generated-srcs
+    # Print the timestamp along with each new line of output, so we can track how long each command takes
+    make go-generated-srcs 2>&1 | while IFS= read -r line; do printf '[%s] %s\n' "$(date '+%Y-%m-%d %H:%M:%S')" "$line"; done
+    git diff --exit-code HEAD
+    { git ls-files --others --exclude-standard ; cat /tmp/untracked ; } | sort | uniq -u >/tmp/untracked-new
+
+    if [[ -s /tmp/untracked-new ]]; then
+        # shellcheck disable=SC2016
+        echo 'Found new untracked files after running `make proto-generated-srcs` and `make go-generated-srcs`. Did you forget to `git add` generated mocks and protos?'
```

Comment:
If you want it to stand out in the logs:
```suggestion
        echo 'ERROR: Found new untracked files after running `make proto-generated-srcs` and `make go-generated-srcs`. Did you forget to `git add` generated mocks and protos?'
```